.travis.yml

```
@@ -6,6 +6,7 @@ php:
     - 7.2
     - 7.3
     - 7.4
+    - '8.0snapshot'
```

```suggestion
    - 8.0
```

We can change this back, now that Travis provides stable 8.0 releases.